floater/utils.py

```diff
@@ -261,6 +261,7 @@ def floats_to_netcdf(input_dir, output_fname,
         df = dd.read_csv(input_path, names=float_columns, dtype=float_dtypes, header=None)
         dfc = df.compute()
         dfcs = dfc.sort_values('npart')
+        step_time = int(step_time)
```

Comment:
@nathanieltarshish Yes, this step here is for the test purpose. It simulates the **type=int** argument in `--step_time` to force **str** input `'86400'` in `test_floats_to_netcdf` into **int**.